release/k3s/k3s.go

```diff
@@ -688,11 +729,13 @@ func (r *Release) CreateRelease(ctx context.Context, client *github.Client, rc b
 
 		if _, err := repository.CreateRelease(ctx, client, opts); err != nil {
 			githubErr := err.(*github.ErrorResponse)
+			logrus.Printf("error: %+v", githubErr)
```

Comment:
Does this need to be logged here?

Reply:
Not really, but it would be good to have some way to see the error. I'm moving it to a debug message.